parlai/mturk/webapp/dev/app.jsx

```diff
@@ -1582,6 +1607,283 @@ class AssignmentPanel extends React.Component {
   }
 }
 
+class ReviewPanel extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      assignments_loading: true, error: false,
+      custom_components: {}, current_worker: null, ordering: 'default',
+      workers_remaining: [], assignments_remaining: 0,
+      assignments_by_worker: {}, current_assignment: null,
+      current_worker_stats: {'approved': 0, 'rejected': 0, 'remain': 0},
+    };
+  }
+
+  parseRawRun(run_data) {
+    // Get reviewable assignments
+    let assignments = run_data.assignments;
+    let assignments_remaining = 0;
+    let reviewable_assigns = assignments.filter(
+      assign => assign.status == 'Reviewable'
+    );
+    // Bucket assignments by worker
+    let assignments_by_worker = {};
+    for (const idx in reviewable_assigns) {
+      let assign = reviewable_assigns[idx];
+      if (assignments_by_worker[assign.worker_id] == undefined) {
+        assignments_by_worker[assign.worker_id] = {
+          assigns: [], bad_feedback: false, worker_id: assign.worker_id,
+        };
+      }
+
+      assignments_remaining += 1;
+      if (!!assign.received_feedback && assign.received_feedback.rating < 3) {
```

Comment:
could be good to standardize these rating values somewhere, so that it's easier to understand what `rating < 3` really means